### PR TITLE
LPFE-16: Impossible to place order

### PR DESCRIPTION
**Pull request by dezoxel**
Fix for: https://github.com/lunches-platform/fe/issues/16

### CODE REVIEW

File: typings.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "globalDependencies": {
     "angular": "github:DefinitelyTyped/DefinitelyTyped/angularjs/angular.d.ts#dc9dabe74a5be62613b17a36053097",
+    "angular-material": "registry:dt/angular-material#1.0.0-rc5.0+20160627174759",
```
Comment (warlock39) [inline typings.json:4]: Is there any stable version of dependency?

File: src/app/models/order.service.ts [hunk 2]
```diff
@@ -67,9 +69,9 @@ export class OrderService {
   }
 
   makeOrder(order: Order) {
-    const url = 'http://dinners/api/order';
-    console.log('POST ' + url);
-    console.log(JSON.stringify(this.prepareOrderForApi(order)));
+    const url = 'http://api.lunches.com.ua/orders';
```
Comment (warlock39) [inline src/app/models/order.service.ts:72]: It would be great to store protocol and host name in some configuration file

File: typings.json (hunk 1 above)
Comment (dezoxel) [inline typings.json:4]: Yep, but current version is enough stable. I haven't seen any issue with that

File: src/app/models/order.service.ts (hunk 2 above)
Comment (dezoxel) [inline src/app/models/order.service.ts:72]: https://github.com/lunches-platform/fe/issues/19

Comment (dezoxel) [inline src/app/models/order.service.ts:72]: Will fix it later. I don't have any stable approach to store configs

File: src/app/containers/basket.component.ts [hunk 3]
```diff
@@ -30,7 +38,25 @@ export class BasketController {
   }
 
   makeOrder(): void {
-    this.lOrderService.makeOrder(this.order);
+    this.lOrderService.makeOrder(this.order)
+      .then(res => {
+        this.showToast('Order has been placed!');
+      })
+      .catch(err => {
+        this.showToast('Error! Unable to place order');
```
Comment (warlock39) [inline src/app/containers/basket.component.ts:47]: If you have possibility, you should output server error into console

File: src/app/containers/basket.component.ts [hunk 4]
```diff
@@ -30,7 +38,25 @@ export class BasketController {
...
+      })
+      .finally(() => {
+        this.$state.go('week-menu');
+      });
+  }
+
+  private showToast(msg: string): void {
```
Comment (warlock39) [inline src/app/containers/basket.component.ts:54]: Adding a toast functionality better to implement in a separate pull request

Comment (dezoxel) [inline src/app/containers/basket.component.ts:54]: I'm using built-in service for toasts from Angular Material. It is already implemented. `showToast` is just a helper method for this particular UI component.

File: src/app/containers/basket.component.ts (hunk 3 above)
Comment (dezoxel) [inline src/app/containers/basket.component.ts:47]: Ok, will fix

**Pull request merged**